tests/test_core/test_core_standalone.py

```diff
@@ -891,3 +832,78 @@ def gb_file(tmp_dir, request):
 def test_gb_suffixes(gb_file):
     seqs = load_unaligned_seqs(gb_file)
     isinstance(seqs, SequenceCollection)
+
+
+@pytest.mark.parametrize(
+    "new_type, seq_type",
+    [(True, new_alignment.SequenceCollection), (False, SequenceCollection)],
+)
+@pytest.mark.parametrize("path_type", [str, pathlib.Path])
+def test_make_unaligned_seqs(new_type, seq_type, path_type):
+    """test SequenceCollection constructor utility function"""
+    data = {"a": "AGGTT", "b": "AG"}
+    got = make_unaligned_seqs(data, moltype="dna", new_type=new_type)
+    assert isinstance(got, seq_type)
+    assert got.to_dict() == data
+    assert got.info["source"] == "unknown"
+
+    # info works
+    got = make_unaligned_seqs(data, moltype="dna", info=dict(a=2))
+    assert got.info["a"] == 2
+
+    # invalid info raises
+    with pytest.raises(AssertionError):
+        _ = make_unaligned_seqs(data, moltype="dna", info=2)
+
+    # source works
+    got = make_unaligned_seqs(
+        data, source=path_type("somewhere"), moltype="dna", new_type=new_type
+    )
+    assert got.info["source"] == "somewhere"
+
+
+@pytest.mark.parametrize(
+    "new_type, seq_type",
+    [(True, new_alignment.SequenceCollection), (False, SequenceCollection)],
+)
+def test_load_unaligned_seqs(new_type, seq_type):
+    """test loading unaligned from file"""
+    path = os.path.join(DATA_DIR, "brca1_5.paml")
+    got = load_unaligned_seqs(path, moltype="dna", new_type=new_type)
+    assert isinstance(got, seq_type)
+    assert "Human" in got.to_dict()
+    assert got.info["source"] == path
+
+
+@pytest.mark.parametrize("new_type", [True, False])
+def test_load_unaligned_seqs_no_format(new_type):
+    """test loading unaligned from file"""
+    with pytest.raises(ValueError):
+        load_unaligned_seqs("somepath", moltype="dna", new_type=new_type)
+
+
+def test_load_seq():
```

Comment:
add a refactor comment to use the pytest `tmp_path` fixture, more robust than tmpdir